classes/book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class Book < Item
+  attr_reader :publisher, :cover_state
+
+  def initialize(publish_date, publisher, cover_state)
+    super(publish_date)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
```

Comment:
- [ ] please kindly add other properties of class Book motioned in the diagram like below
 attr_accessor :title, :publisher, :cover_state, :label
  def initialize(title, publish_date, publisher, cover_state, label: [])
    super(publish_date)